WebRoot/public/extuserprov.jsp

```diff
@@ -147,5 +151,10 @@
 		return !isError;
 	}
 </script>
+<% if (request.getAttribute("errorCode") != null) { %>
+	<script>
+		document.getElementById("ZLoginErrorPanel").style.display = "block";
```

Comment:
can we move this to line 66, we are already defining display: none there can we conditionally set it?